ast.go

```diff
@@ -246,47 +230,65 @@ func (node *Subquery) ContainsSubquery() bool {
 	return true
 }
 
-// ParenTableExpr represents a parenthesized list of TableExpr.
+// ParenTableExpr represents a parenthesized TableExpr.
 type ParenTableExpr struct {
-	TableExprList TableExprList
+	TableExpr TableExpr
 }
 
 // String returns the string representation of the node.
 func (node *ParenTableExpr) String() string {
-	return fmt.Sprintf("(%v)", node.TableExprList.String())
+	return fmt.Sprintf("(%v)", node.TableExpr.String())
+}
+
+// JoinOperator represents a join operator.
+type JoinOperator struct {
+	Op      string
+	Natural bool
```

Comment:
Natural and Outer flags to indicate the presence of the NATURAL and OUTER keywords